releasetasks/templates/release_graph.yml.tmpl

```diff
@@ -65,6 +65,19 @@ tasks:
     {% endif %}
 
     {% if verifyConfigs %}
+        {% macro updateVerify_task(platform) %}
+            {% if platform in ["linux", "linux64"] %}
```

Comment:
ah, gotcha. this is just the definition. I guess I should have read down a few lines ;)